core/src/main/resources/hudson/model/AbstractBuild/index.jelly

```diff
@@ -56,7 +56,7 @@ THE SOFTWARE.
         <t:editableDescription permission="${it.UPDATE}" />
       </div>
 
-      <table style="margin-top: 1em; margin-left:1em;">
+      <div>
```

Comment:
Not sure how things are done with styling etc now in the modern times. But I think it would be nice if this div had an `id` for page decorators et.al. if they need to manipulate it somehow. Themes would also have it easier to style it I guess?

Reply:
not really sure why you would need them.

themes should be able to use selectors and if they are finding a need for it they can add them at the time.